src/components/EditDisplayName/EditDisplayName.module.scss

```diff
@@ -21,4 +21,8 @@
 
 .successMessage {
   margin-top: 8px;
+
```

Comment:
I think this is covered in the alerts story now. I don't think we need it? Lmk if I'm missing something.

Reply:
That makes sense! I'll revert this change 👍